.obs/specfile/elemental-toolkit.spec

```diff
@@ -49,7 +49,7 @@ BuildRequires:  golang-packaging
 %global commit     d1ae3f9a425de2618f9058f3b37583ef3ce52c7d
 %gometa
 %if (0%{?centos_version} == 800) || (0%{?rhel_version} == 800)
-BuildRequires:  go1.20
+BuildRequires:  go1.22
```

Comment:
<s>This might not be available in SLE Micro 6 🤔 </s>

Should become available 🤞🏻